src/main/java/org/checkerframework/specimin/JavaTypeCorrect.java

```diff
@@ -29,7 +30,7 @@ class JavaTypeCorrect {
    * This map is for type correcting. The key is the name of the current incorrect type, and the
    * value is the name of the desired correct type.
    */
-  private Map<String, String> typeToChange;
+  private Map<@ClassGetSimpleName String, @ClassGetSimpleName String> typeToChange;
```

Comment:
It makes me very nervous that these are simple names, given our discussion just a few minutes ago about programs that use two classes with the same simple name but different fully-qualified names. For now, this is fine, but we should keep this in mind as we're testing to see if that example works correctly.